mbuild/compound.py

```diff
@@ -1727,15 +1728,18 @@ def _energy_minimize_openbabel(self, tmp_dir, steps=1000, algorithm='cg',
         """
 
         openbabel = import_('openbabel')
-        md = import_('mdtraj')
-        from mdtraj.core.element import get_by_symbol
         for particle in self.particles():
-            try:
-                get_by_symbol(particle.name)
-            except KeyError:
-                raise MBuildError("Element name {} not recognized. Cannot "
-                                  "perform minimization."
-                                  "".format(particle.name))
+            if particle.element is None:
+                try:
+                    element_from_symbol(particle.name[:2])
```

Comment:
What is this slice for?

Reply:
If there is not element attribute I am trying to infer the element symbol from the first two characters of the compound name. The `ele.element_from_symbol` requires an exact match.

Now that you point it out I would probably prefer to not slice and require an exact match. Else you run the risk of `Compound` turning into `Cobalt` and `Carbon` turning into `Calcium`. Yeah good catch, that is dumb :).